### PR TITLE
Add url to build artifacts

### PR DESCRIPTION
**Pull request by akharche**

### CODE REVIEW

File: common/mediasdk_directories.py [hunk 1]
```diff
@@ -166,6 +168,20 @@ def get_commit_dir(cls, branch, build_event, commit_id):
 
         return pathlib.Path(cls._builds_root_path) / branch / build_event / commit_id
 
+    @classmethod
+    def get_artifact_url(cls, dir):
```
Comment (dlobanox) [inline common/mediasdk_directories.py]: Need to do the same function for closed source.
Build runner will fail "copy" stage without it.

**Comment by adydychk** [comment]
It should be tested on AUX instance before merging.

File: ted_adapter/test_adapter.py [hunk 2]
```diff
@@ -119,6 +119,9 @@ def copy_logs_to_share(self):
         rotate_dir(self.tests_artifacts_dir)
         print(f'Copy results to {self.tests_artifacts_dir}')
 
+        tests_artifacts_url = MediaSdkDirectories.get_artifact_url(self.tests_artifacts_dir)
```
Comment (azhogov) [inline ted_adapter/test_adapter.py]: different API for tests will be needed

File: common/mediasdk_directories.py [hunk 3]
```diff
@@ -55,9 +55,10 @@ class MediaSdkDirectories(object):
     """
     Container for static links
     """
-    _tests_root_path = r'/media/tests'
-    _builds_root_path = r'/media/builds'
-    _root_path = r'/media'
+    _mount_point_root_dir = r'/media'
+    _tests_root_path = pathlib.Path(_mount_point_root_dir) / 'tests'
```
Comment (adydychk) [inline common/mediasdk_directories.py]:
```
 _mount_point_root_dir = pathlib.Path('/media')
 _tests_root_path = _mount_point_root_dir / 'tests'
...
```

**Pull request merged**